mirage/config.js

```diff
@@ -73,7 +73,7 @@ export default function () {
   });
 
   this.get('/users/permissions', (schema, request) => {
-    let authorization = request.requestHeaders.Authorization;
+    const { authorization } = request.requestHeaders;
```

Comment:
Hmm.. Are you sure about this change? Before it was with capital `A` (and JS is case sensitive). Just wanted to bring it to your attention.

Reply:
Thanks for bringing it to my attention :) This change is due to the fact that `fetch` lowercases header names

https://github.com/ember-cli/ember-fetch/issues/100
https://github.com/github/fetch/issues/249

This resulted in an issue where some tests were failing because this function was now receiving the header as `authorization` rather than `Authorization`, so the change is intentional. Although, now that I think of it, maybe it would be safer to do like an `authorization || Authorization` approach. What do you think?

Reply:
Ah, I see! Thanks for clarifications, I didn't know that :) To be honest, this behavior of fetch is kinda weird - not all backends are case insensitive.

In this case, let's keep it lowercase, so that we had in tests exactly the same as in production 👍

Reply:
Sounds good :) Yeah I agree that it's weird, and was quite surprised when I found out about it. A lot of people might have limited control over their backend, or updating the backend for it might take a huge amount of work. But it seems like the preferred way forward